Library/Homebrew/rubocops/lines.rb

```diff
@@ -938,9 +938,7 @@ def audit_formula(formula_nodes)
             end
           end
 
-          # TODO: Enforce order of dependency types so we don't need to check for
-          #       depends_on "rustup" => [:test, :build]
-          [:build, [:build, :test], [:test, :build]].each do |type|
+          [:build, [:build, :test]].each do |type|
```

Comment:
```suggestion
          [:build, [:build, :test], [:test, :build]].each do |type|
```
If we don't want to write a cop for it then we should retain the last case to prevent this dependency from sneaking in from a
```ruby
depends_on "rustup" => [:test, :build]
```
line.